app/src/components/common/navbar/NavBar.js

```diff
@@ -1,32 +1,26 @@
-import Link from "next/link";
+import NavLink from "../link/NavLink";
```

Comment:
@

Reply:
well noted, thank you!
All the imports were reviewed and corrected